packages/ember-views/lib/views/view.js

```diff
@@ -1345,15 +1345,21 @@ var View = CoreView.extend(
   scheduleRevalidate(node, label, manualRerender) {
     if (node && !this._dispatching && node.guid in this.env.renderedNodes) {
       if (manualRerender) {
-        Ember.deprecate(`You manually rerendered ${label} (a parent component) from a child component during the rendering process. This rarely worked in Ember 1.x and will be removed in Ember 2.0`);
+        Ember.deprecate(`You manually rerendered ${label} (a parent component) from a child component during the rendering process. This rarely worked in Ember 1.x and will be removed in Ember 2.0`,
+                        false,
+                        { id: 'ember-views.manual-parent-rerender', until: '3.0.0' });
```

Comment:
it is already changed in master, and should be changed to 3.0 here https://github.com/emberjs/ember.js/blob/341a325a13fc685a10399de600586b6fb744e99f/packages/ember-views/lib/mixins/view_support.js#L682